src/main/java/com/fasterxml/jackson/databind/deser/std/EnumDeserializer.java

```diff
@@ -172,16 +172,15 @@ public static JsonDeserializer<?> deserializerForCreator(DeserializationConfig c
      * @since 2.8
      */
     public static JsonDeserializer<?> deserializerForCreator(DeserializationConfig config,
-            Class<?> enumClass, AnnotatedMethod factory,
```

Comment:
Ok, need to keep the old factory method, make it `@Deprecated`; add new one (with `@since 2.19`)

Reply:
I can do this.